pkg/flowkit/services/tests.go

```diff
@@ -84,13 +84,17 @@ func (t *Tests) importResolver(scriptPath string, readerWriter flowkit.ReaderWri
 			return "", fmt.Errorf("cannot import from %s", location)
 		}
 
-		importedContract, err := t.resolveContract(stringLocation)
-		if err != nil {
-			return "", err
+		relativePath := stringLocation.String()
+		for _, contract := range *t.state.Contracts() {
+			if !strings.Contains(relativePath, contract.Location) {
+				return "", fmt.Errorf(
+					"cannot find contract with location '%s' in configuration",
+					relativePath,
+				)
+			}
```

Comment:
Why does it need to terminate the loop on the first failure? wouldn't it need to search for all contracts?

Sorry, for the very late comment/question, just noticed this only now.

Reply:
That's a great catch Supun :100: :100: 
I have fixed this in https://github.com/onflow/flow-cli/pull/985/commits/7dd7322c770da105aebed9dc133e57a0004ee7b3